aquarius/app/assets.py

```diff
@@ -767,3 +849,49 @@ def _reorder_services(services):
             result.append(service)
 
     return result
+
+def getsigneraddress(message, signature):
+    '''
+    Get signer address of a previous signed message
+    :param str message: Message
+    :param str signature: Signature obtain with web3.eth.personal.sign
+    :return: Address or None in case of error
+    '''
+    try:
+        logger.debug('got %s as a message' % message)
+        message_hash = defunct_hash_message(text=message)
+        logger.debug('got %s as a message_hash' % message_hash)    
+        address_recovered = web3.eth.account.recoverHash(message_hash, signature=signature)
+        logger.debug('got %s as address_recovered' % address_recovered)    
+        return address_recovered
+    except Exception as e:
+        logger.error(e)
+        return None  
+
+def compare_eth_addresses(address, checker):
```

Comment:
This module should only have the endpoints, all other function can be moved to a helper/util module.

Reply:
fixed